src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,8 +14,12 @@
     private Phone phone;
     private Email email;
     private Address address;
-
+    private int sequenceNumber;
+    
     private final UniqueTagList tags;
+    
+    private static int nextSequenceNumber = 0;
```

Comment:
Also, as mentioned in tutorial ~ build/test your code after you've done implementations (that's the very basic of SE), since the above code would only work with "1" not "0".